### PR TITLE
Implementation preview import and config result command

### PR DESCRIPTION
**Pull request by prashanthlgowda**
Implementation preview import and config result command

### CODE REVIEW

**Review by muqeethkowkab — APPROVED** [review]
Changes Looks Good.

**Comment by michaelsteven** [comment]
this should wait on merge of PR in smi-lib-wsman before merging this one.

**Review by muqeethkowkab — APPROVED** [review]
Changes Looks Good.

**Pull request merged**
